package proto_ff;

import "yd_fieldoptions.proto";


message E_GodevilPreposetask
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "序号ID"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 m_modeltype = 3[(yd_fieldoptions.field_cname) = "所属阶段"];
	optional int32 m_starttask = 4[(yd_fieldoptions.field_cname) = "任务ID"];
	optional int32 m_endtask = 5[(yd_fieldoptions.field_cname) = "结束任务ID"];
	optional int32 m_condition = 6[(yd_fieldoptions.field_cname) = "条件"];
	optional int32 m_value = 7[(yd_fieldoptions.field_cname) = "条件值"];
	optional int32 m_rewardid = 8[(yd_fieldoptions.field_cname) = "奖励ID"];
}


message Sheet_GodevilPreposetask
{
	repeated E_GodevilPreposetask E_GodevilPreposetask_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_GodevilGodevillvAttributeDesc
{
	optional int32 m_type = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 m_value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message E_GodevilGodevillv
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "等级"];
	optional int64 m_exp = 2[(yd_fieldoptions.field_cname) = "下级经验"];
	optional int64 m_explimit = 3[(yd_fieldoptions.field_cname) = "经验上限"];
	optional int32 m_itemid = 4[(yd_fieldoptions.field_cname) = "突破材料"];
	optional int32 m_num = 5[(yd_fieldoptions.field_cname) = "材料数量"];
	repeated E_GodevilGodevillvAttributeDesc m_attribute = 6[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 7];
}


message Sheet_GodevilGodevillv
{
	repeated E_GodevilGodevillv E_GodevilGodevillv_List = 1[(yd_fieldoptions.field_arysize)=40];
}

message E_GodevilGodevilprivilege
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_oldprice = 2[(yd_fieldoptions.field_cname) = "原价元"];
	optional int32 m_newprice = 3[(yd_fieldoptions.field_cname) = "现价"];
	optional int32 m_rechargid = 4[(yd_fieldoptions.field_cname) = "计费点"];
	optional int32 m_type = 5[(yd_fieldoptions.field_cname) = "特权类型"];
	optional int32 m_lv = 6[(yd_fieldoptions.field_cname) = "特权等级"];
	optional int32 m_functiontype = 7[(yd_fieldoptions.field_cname) = "功能类型"];
	optional int32 m_parama = 8[(yd_fieldoptions.field_cname) = "参数"];
	optional int32 m_equipmentid = 9[(yd_fieldoptions.field_cname) = "获取装备道具ID"];
}


message Sheet_GodevilGodevilprivilege
{
	repeated E_GodevilGodevilprivilege E_GodevilGodevilprivilege_List = 1[(yd_fieldoptions.field_arysize)=40];
}
